modules/retail/pages/retail-offline.adoc

```diff
@@ -37,9 +37,27 @@ For more about kernel parameters, see xref:salt:formula-pxe.adoc[].
 
```

Comment:
Because this update still needs a connection to the SUMA, it just isn't that bandwidth intensive, it is wrong to call it 'Offline'. Can you change it to 'Cached Terminal Updates' or 'Preloaded Terminal Updates'.

Reply:
In the now existing text, we mostly use the term "Cached".  Thus I'd prefer it for the heading.  I changed my local copy.